src/infer_response.h

```diff
@@ -72,9 +73,10 @@ class InferResponse {
  public:
   InferResponse(
       const std::vector<std::shared_ptr<PbTensor>>& output_tensors,
-      std::shared_ptr<PbError> error = nullptr,
+      std::shared_ptr<PbError> error = nullptr, std::string parameters = "",
       const bool is_last_response = true, void* id = nullptr);
   std::vector<std::shared_ptr<PbTensor>>& OutputTensors();
+  std::string& Parameters();
```

Comment:
Looks like this parameters string is expected to be JSON serializable - can you add some docstrings describing that expectation?

Reply:
Yes, added the comment: [Mark response parameters accessor const and JSON serializable](https://github.com/triton-inference-server/python_backend/pull/394/commits/f23114704d0a6135962b72ccb88b55425c48b5b0)